templates/env.rc

```diff
@@ -63,17 +63,30 @@ CAPO_OPENSTACK_CLOUD_YAML_CONTENT=$(cat "${CAPO_CLOUDS_PATH}")
 
 CAPO_YQ_VERSION=$(yq -V)
 yqNavigating(){
-        if [[ ${CAPO_YQ_VERSION} == *"version 1"* || ${CAPO_YQ_VERSION} == *"version 2"* || ${CAPO_YQ_VERSION} == *"version 3"* ]]; then
-                yq r $1 $2
-        else
-                yq e .$2 $1
-        fi
+  if [[ ${CAPO_YQ_VERSION} == *"version 1."* || ${CAPO_YQ_VERSION} == *"version 2."* || ${CAPO_YQ_VERSION} == *"version 3."* ]]; then
```

Comment:
Is there a reason for adding the dots here? If so, is there any reason not to do it elsewhere?

Reply:
No, not really. I just reverted the change